infinispan-remote/reactive-api/src/main/java/org/infinispan/tutorial/simple/reactive/InfinispanReactiveApi.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.tutorial.simple.reactive;
+
+import org.infinispan.api.Infinispan;
+import org.infinispan.api.mutiny.MutinyCache;
+import org.infinispan.hotrod.configuration.ClientIntelligence;
+import org.infinispan.hotrod.configuration.HotRodConfigurationBuilder;
+
+import java.time.Duration;
+
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.HOST;
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.PASSWORD;
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.SINGLE_PORT;
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.TUTORIAL_CACHE_CONFIG;
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.TUTORIAL_CACHE_NAME;
+import static org.infinispan.tutorial.simple.connect.TutorialsConnectorHelper.USER;
+
+public class InfinispanReactiveApi {
+
+   public static void main(String[] args) {
+      // New API Connection
+      HotRodConfigurationBuilder builder = new HotRodConfigurationBuilder();
+      builder
+            .clientIntelligence(ClientIntelligence.BASIC)
+            .addServer().host(HOST).port(SINGLE_PORT)
+            .security().authentication()
+            .username(USER)
+            .password(PASSWORD);
+      builder.remoteCache(TUTORIAL_CACHE_NAME)
+            .configuration(TUTORIAL_CACHE_CONFIG.replace("CACHE_NAME", TUTORIAL_CACHE_NAME));
+
+      Infinispan infinispan = Infinispan.create(builder.build());
+      MutinyCache<String, String> cache = infinispan.mutiny()
+            .caches().<String, String>get(TUTORIAL_CACHE_NAME)
+            .await().atMost(Duration.ofMillis(100));
```

Comment:
```suggestion
      MutinyCache<String, String> cache = infinispan.mutiny()
            .caches().<String, String>get(TUTORIAL_CACHE_NAME)
            .onItem().delayIt().by(Duration.ofSeconds(1))
            .await().atMost(Duration.ofSeconds(2));
```

Without waiting, the first access can proceed and execute operations without the cache in place. I'll look into that, it is not something that I would expect.